src/config/serverConfigDefaults.ts

```diff
@@ -42,6 +42,8 @@ const ServerConfigDefaults: Partial<IServerConfig> = {
     isoformOverrideSource: 'uniprot',
     show_hotspot: true,
     show_oncokb: true,
+    show_cbioportal: true,
```

Comment:
this is obviously a long standing issue but we need to at very least improve specificity of propery names.  like, oncoprint_show_cbioportal_annotations?